setup/gcp/installer/forseti_installer.py

```diff
@@ -0,0 +1,291 @@
+# Copyright 2017 The Forseti Security Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+""" Forseti Installer"""
+
+from __future__ import print_function
+from abc import ABCMeta
+from abc import abstractmethod
+
+import sys
+
+from util.utils import (
+    print_banner, get_forseti_version, format_service_acct_id,
+    infer_version)
+from util.constants import (
+    FORSETI_CONF_PATH, DEPLOYMENT_TEMPLATE_OUTPUT_PATH,
+    MESSAGE_DEPLOYMENT_HAD_ISSUES, MESSAGE_DEPLOYMENT_TEMPLATE_LOCATION,
+    MESSAGE_VIEW_DEPLOYMENT_DETAILS, MESSAGE_FORSETI_CONFIGURATION_GENERATED,
+    MESSAGE_FORSETI_CONFIGURATION_GENERATED_DRY_RUN, DEFAULT_BUCKET_FMT_V2,
+    MESSAGE_FORSETI_BRANCH_DEPLOYED, MAXIMUM_LOOP_COUNT)
+from util.gcloud import (
```

Comment:
please fix this kind of method importing everywhere

Reply:
Updated.